test/ez-field-edit.html

```diff
@@ -0,0 +1,39 @@
+<!doctype html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, minimum-scale=1, initial-scale=1, user-scalable=yes">
+
+        <title>ez-field-edit test</title>
+
+        <script src="../../webcomponentsjs/webcomponents-lite.js"></script>
+        <script src="../../web-component-tester/browser.js"></script>
+
+        <link rel="import" href="../ez-field-edit.html">
+    </head>
+    <body>
+        <test-fixture id="BasicTestFixture">
+            <template>
+                <ez-field-edit>
+                    <label>Field label</label>
+                    <input id="input1" type="text" pattern="\d*"/>
+                </ez-field-edit>
+            </template>
+        </test-fixture>
+        <test-fixture id="BasicTestFixtureMultipleInputs">
+            <template>
+                <ez-field-edit>
+                    <div id="lowestCommonParent1">
+                        <label>Field label</label>
```

Comment:
related to comment about search the common ancestor, those fixtures are too simple. Having the `label` and the `input` with the same parent node is clearly an over simplification